.github/workflows/test-cm-based-submission-generation.yml

```diff
@@ -17,14 +17,13 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         python-version: [ "3.12" ]
-        division: ["closed", "open"]
+        division: ["closed", "open", "open-closed"]
```

Comment:
closed-open ?

Reply:
modified in commit: https://github.com/mlcommons/cm4mlops/pull/535/commits/d785cc0ac2fa868d527a8ca0fe6f4b69b16e929e